mycity/mycity/intents/trash_intent.py

```diff
@@ -1,26 +1,35 @@
 """
 Functions for Alexa responses related to trash day
+
 """
-from mycity.intents import intent_constants
-from mycity.intents.custom_errors import \
-    InvalidAddressError, BadAPIResponse, MultipleAddressError
-from mycity.intents.user_address_intent import clear_address_from_mycity_object
-import mycity.intents.speech_constants.trash_intent as speech_constants
-from mycity.mycity_response_data_model import MyCityResponseDataModel
-import mycity.utilities.address_utils as address_utils
-from streetaddress import StreetAddressParser
 
 import re
-import requests
 import logging
+import typing
+

```

Comment:
extra newline?

Reply:
Separating stdlib from package from local imports. If we decide it's too much because we have an average of 5 imports then sure.